src/libraries/Microsoft.Extensions.Configuration.Binder/tests/Common/ConfigurationBinderTests.Helpers.cs

```diff
@@ -0,0 +1,154 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Extensions
+#if BUILDING_SOURCE_GENERATOR_TESTS
+    .SourceGeneration
+#endif
+    .Configuration.Binder.Tests
+{
+    public static class TestHelpers
+    {
+        public static bool NotSourceGenMode
```

Comment:
Would it be possible to share a couple of representative generated source files produced by the tests? I think it might make reviewing a bit easier.

Reply:
Here's the current output for the test suite: all applicable calls (Get, Bind, Configure) are included: https://gist.github.com/layomia/31bd3ed8b09dbf1c2244f6e227596d04.

Reply:
Can we add a handful baseline tests? I've found it to be EXTREMELY helpful in reviewing and diffing generated code as we go.

See 

https://github.com/dotnet/runtime/blob/0488519e709811fd6f96aa0aed4c17a72c00898c/src/libraries/Microsoft.Extensions.Logging.Abstractions/tests/Microsoft.Extensions.Logging.Generators.Tests/LoggerMessageGeneratorEmitterTests.cs#L36-L48

https://github.com/dotnet/runtime/blob/f76b2b1e4d2a4127086b863213b1575d2b6696d6/src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexGeneratorOutputTests.cs#L14-L20

https://github.com/dotnet/aspnetcore/blob/f09180f1154a47f1e1dc9a9b46130ef058248a85/src/Http/Http.Extensions/test/RequestDelegateGenerator/RequestDelegateGeneratorTests.cs#L38-L60

We don't need to baseline every test. But just a small subset to ensure the readability/formatting and ease the code reviewing.